buildlib/fedora.Dockerfile

```diff
@@ -20,6 +20,14 @@ RUN dnf install -y \
     make \
     maven \
     numactl-devel \
+    python \
     rdma-core-devel \
     rpm-build \
     && dnf clean dbcache packages
+RUN mkdir -p /etc/clang
+RUN export BUILD_ROOT=/tmp/llvm-project && \
+    git clone https://github.com/gleon99/llvm-project.git --depth=1 -b master --single-branch ${BUILD_ROOT} && \
```

Comment:
i don't see the clang-format changes in this llvm fork

Reply:
The changes are there, in the master branch:
https://github.com/gleon99/llvm-project/commit/e893bbd9efe3eb29c97548a9e268a1eb625c2bb3